crates/aptos-crypto/src/bls12381/bls12381_sigs.rs

```diff
@@ -113,6 +113,18 @@ impl Signature {
 
         self.verify_aggregate_arbitrary_msg(&msgs_refs, pks)
     }
+
+    /// Return a dummy signature.
+    #[cfg(any(test, feature = "fuzzing"))]
+    pub fn dummy_signature() -> Self {
+        // TODO: maybe Alin knows better way to generate a dummy signature
```

Comment:
Well, since you've set the dummy private key equal to 1, the signature will just be H(m), which I think you could compute directly via [blst_encode_to_g1.html](https://docs.rs/blst/latest/blst/fn.blst_encode_to_g1.html), but I think what you have here is best. Exponentiation should be fast when the private key is 1.

Maybe that's why you picked it as 1?